opencti-platform/opencti-front/src/private/components/techniques/courses_of_action/CourseOfActionCreation.tsx

```diff
@@ -199,18 +200,20 @@ export const CourseOfActionCreationForm: FunctionComponent<CourseOfActionFormPro
         setFieldValue,
         values,
       }) => (
-        <Form>
+        <Form style={{ margin: '20px 0 20px 0' }}>
```

Comment:
Please remove this margin
```suggestion
        <Form>
```

Reply:
@frapuks - Corrected